spec/controllers/api/v0/business_days_controller_spec.rb

```diff
@@ -9,6 +9,7 @@
     end
 
     it 'should handle holidays correctly' do
+      BusinessTime::Config.holidays << Date.parse('2016-09-05')
```

Comment:
yeah I really think we should do it separately so we can have a clean commit history to understand why this change was made. I will do it and we can merge that first so other things arent broken